app/frontend/src/search/ui/input/keyword.test.js

```diff
@@ -0,0 +1,19 @@
+import { onSubmit } from './keyword';
+
+describe('keyword search box', () => {
+
+    const client = {};
+    let performSearch = null;
+
+    beforeEach(() => {
+        client.refresh = jest.fn();
+        performSearch = jest.spyOn(client, 'refresh');
+    });
+
+    describe('onSubmit', () => {
+        test('does not performs search if unsuccesful', () => {
+            onSubmit(client);
+            expect(performSearch).toHaveBeenCalledTimes(1);
```

Comment:
Confused that the test description says 'does not perform search' but we expect the search to have been performed once.